CoreLocationDemo/MapKitViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MapKitViewController.swift
+//  CoreLocationDemo
+//
+//  Created by MCS on 12/20/18.
+//  Copyright © 2018 Jeremiah Hawks. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class MapKitViewController: UIViewController {
+    var initialLocation: CLLocation?
+    let regionRadius : CLLocationDistance = 1000
+    let annotation = MKPointAnnotation()
+
+    //MARK: Outlets
+    @IBOutlet weak var mapView: MKMapView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        
+        // Do any additional setup after loading the view.
+        LocationManager.shared.getLastLocation  { (optionalPlacemark) in
+
+            let placemark = optionalPlacemark
+
+            let getLatitud = placemark?.location?.coordinate.latitude
```

Comment:
I would like to see better names for these. just latitude and longitude. 

Also, I would like to see them safely unwrapped. This is more for your teammates, and less for you. You wrote the code so you understand it and know how to use it safely. If a teammate is trying to write a feature that integrates with your code, they may not take the time to understand, and may use it unsafely. Its almost always better to safely unwrap, or use optional binding or something like that.